CHANGELOG.md

```diff
@@ -5,13 +5,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- BREAKING: Required configuration of validation engine
```

Comment:
please add a upgrade guide detailing all breaking changes and how to update (+ linking to the dry upgrade guide if there is one)

Reply:
Isn't this sufficient? https://github.com/runtastic/request_handler/pull/43/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
I somehow missed that section in the readme